lib/grape-swagger/entity/parser.rb

```diff
@@ -35,7 +35,12 @@ def extract_params(exposure)
             memo.merge!(extracted_params)
           else
             opts = value.send(:options)
-            opts[:as] ? memo[Alias.new(value.attribute, opts[:as])] = opts : memo[value.attribute] = opts
+            if opts[:as]
+              memo.delete_if { |k, _| k.is_a?(Alias) && k.renamed == opts[:as] }
+              memo[Alias.new(value.attribute, opts[:as])] = opts
+            else
+              memo[value.attribute] = opts
+            end
```

Comment:
This fixes a bug when the same alias is used multiple times, e.g.:
```ruby
expose :foo, as: :alias
expose :bar, as: :alias
```

With this fix, the last alias takes precedence, which is consistent with the non-aliased behavior.